.yarnrc.yml

```diff
@@ -1,11 +1,17 @@
 nodeLinker: node-modules
 
+enableGlobalCache: false
+
+nmMode: hardlinks-local
+
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
     spec: '@yarnpkg/plugin-interactive-tools'
 
 defaultSemverRangePrefix: ''
 
+compressionLevel: 0
```

Comment:
the zip isn't compressed. More work for action/cache restore / network / decompress (zstd detects when it can compress or not so... now the *.zip files will be compressable by action/cache -> slower but more compressed too)

My previous benchmarks

| CI Scenario             | Install | CI fetch cache | CI persist cache | Setup | 
|-------------------------|--------:|---------------:|-----------------:|------:|
| yarn4 mixed-compression |    ±42s |            ±3s |          *(±9s)* |    0s |
| yarn4 no compression    |    ±34s |            ±4s |          *(±6s)* |    0s |
| pnpm8                   |    ±19s |            ±8s |         *(±29s)* |    2s |

See https://github.com/belgattitude/compare-package-managers